build/run-in-docker

```diff
@@ -20,8 +20,10 @@
 
 set -e
 
-REPODIR=$(git rev-parse --path-format=absolute --show-toplevel)
-GIT_COMMON_DIR=$(git rev-parse --path-format=absolute --git-common-dir)
+REPODIR_REL=$(git rev-parse --show-toplevel)
+REPODIR=$(readlink -e "$REPODIR_REL")
+GIT_COMMON_DIR_REL=$(git rev-parse --git-common-dir)
+GIT_COMMON_DIR=$(readlink -e "$GIT_COMMON_DIR_REL")
```

Comment:
Any reason not to use `realpath` here? Not all platforms that provide readlink support the `-e` option, but many have `realpath` which seems to do what we want here without additional arguments.

Reply:
I was not familiar with`realpath` for some reason. Thanks for pointing to it. Now I am noticing that the man page for GNU coreutils readlink even states:

> Note realpath(1) is the preferred command to use for canonicalization functionality.

Similarly not all versions of realpath support `-e`. However, we can rely on git that the path exists.